web/src/engine/osk/src/visualKeyboard.ts

```diff
@@ -1360,8 +1369,10 @@ export default class VisualKeyboard extends EventEmitter<EventMap> implements Ke
     if (activeKeyboard != null && typeof (activeKeyboard.oskStyling) == 'string')  // KMEW-129
       customStyle = customStyle + activeKeyboard.oskStyling;
 
-    this.styleSheet = createStyleSheet(customStyle); //Build 360
-    this.styleSheetManager.linkStylesheet(this.styleSheet);
+    if(customStyle) {
+      this.styleSheet = createStyleSheet(customStyle); //Build 360
+      this.styleSheetManager.linkStylesheet(this.styleSheet);
+    }
```

Comment:
Highlighting this section, as it prevents the scenario addressed by the other noted block.